src/ocbs/serve.py

```
@@ -1,256 +1,914 @@
+                        # Validate token
+                        serve_record = server_instance._validate_token(token)
+                        if not serve_record:
+                            self.send_error(404, "Invalid or expired token")
+                            return
+                        
+                        # Try to restart the gateway gracefully
+                        try:
+                            import subprocess
+                            # Check if we can use systemctl
+                            result = subprocess.run(
+                                ['which', 'systemctl'],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)